configs/ld/readme.md

```diff
@@ -0,0 +1,29 @@
+# Localization Distillation for Object Detection
+
+## Introduction
+
+LD is the extension of knowledge distillation on localization task, which utilizes the learned bbox distributions to transfer the localization dark knowledge from teacher to student.
+
+- [Localization Distillation for Object Detection](https://arxiv.org/abs/2102.12252)
+
+```latex
+@Article{zheng2021LD,
+  title={Localization Distillation for Object Detection},
+  author= {Zhaohui Zheng, Rongguang Ye, Ping Wang, Jun Wang, Dongwei Ren, Wangmeng Zuo},
+  journal={arXiv:2102.12252},
+  year={2021}
+}
+```
+
+### GFocalV1 with LD
+
+|  Teacher  | Student | Training schedule | Mini-batch size | AP (val) | AP50 (val) | AP75 (val) | AP (test-dev) | AP50 (test-dev) | AP75 (test-dev) | AR100 (test-dev) |
```

Comment:
Suggest add a new column to link each row to the config links. See the config column in [faster_rcnn](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn).